recipe/meta.yaml

```diff
@@ -7,9 +7,12 @@ package:
 source:
   url: https://github.com/docker/docker-py/archive/{{ version }}.tar.gz
   sha256: 0035772df94a86e8db02aac363393755584383aacf50d37b0d55ea817149e1f4
+  patches:
+    - 0001-loosen-pywin32-pin.patch
+    - 0002-update-test-skips.patch
```

Comment:
Are these upstream or upstream-able? If the former please add a link to the commit or PR that added them. If the latter than consider sending them upstream.